api/v1beta2/foundationdbcluster_types.go

```diff
@@ -565,25 +565,25 @@ func FilterByCondition(processGroupStatus []*ProcessGroupStatus, conditionType P
 // groups with that condition will be returned. If a condition is mapped to
 // false in the conditionRules map, only process groups without that condition
 // will be returned.
-func FilterByConditions(processGroupStatus []*ProcessGroupStatus, conditionRules map[ProcessGroupConditionType]bool, ignoreRemoved bool) []string {
+func FilterByConditions(processGroups []*ProcessGroupStatus, conditionRules map[ProcessGroupConditionType]bool, ignoreRemoved bool) []string {
 	result := make([]string, 0)
```

Comment:
Changes here are mainly done to increase readability.

Reply:
We use the name `processGroupStatus` in multiple places what is the benefit of changing it only in this place? If we see a benefit in changing the name of the variable we should do it consistently in all methods. I would prefer having a dedicated PR for that, since those changes are out-of-scope for the referenced issue.